.firebaserc

```diff
@@ -1,5 +1,6 @@
 {
   "projects": {
-    "default": "goplan-3b4b1"
+    "default": "goplan-3b4b1",
+    "dev": "goplan-3b4b1"
```

Comment:
where do we run the function deploy command?  inside the root directory of the main project or inside `function/`?

Reply:
inside /function , i will add instructions

Reply:
added in Readme